### ISSUE TITLE
Visualize mempool (pending transactions)

### ISSUE THREAD

**Issue by davidyuk** [aeternity/aescan#725]
There is an endpoint [/debug/transactions/pending] returning transactions that have not been mined yet. Currently, it is almost empty, but there can be stuck transactions because of increased gas price. I propose to add a page showing aggregated data on these transactions, or transactions by themself (since we don't have much data yet). Transactions should be ordered by gas price, split by block size (in gas). Also put invalid transactions to the end.

Inspired by https://mempool.space/

[/debug/transactions/pending]: https://testnet.aeternity.io/v3/debug/transactions/pending

**Comment by janmichek** [comment]
> gas price, split by block size (in gas

@davidyuk do you have idea how to get data from debug endpoint which is NOT meant to be public?

I discussed this over slack, and pending transactions ar hidden for some reason.

Summary of the discussion here https://github.com/aeternity/aescan/issues/860

**Comment by davidyuk** [comment]
Let's find somebody who will setup a debug access for aescan backend. Not needed to close the issue just because of this.

**Comment by janmichek** [comment]
> Let's find somebody who will setup a debug access for aescan backend. Not needed to close the issue just because of this.

@davidyuk Can we resume on this one, please?

**Comment by davidyuk** [comment]
With Dincho we decided that mdw is a good place to expose pending transactions, and it looks like the mdw team has already picked up the task.

@yaboiishere can you estimate how much time it will take to implement https://github.com/aeternity/ae_mdw/issues/1950?

**Comment by yaboiishere** [comment]
Forgot to update :sweat_smile: It has been merged to master, so it should be usable in the next release

**Comment by janmichek** [comment]
> Forgot to update 😅 It has been merged to master, so it should be usable in the next release

And so what is the URL pls? Will this be documented in docs?

**Comment by yaboiishere** [comment]
Yes, there are 2 endpoints `v3/transactions/pending` and `v3/transactions/pending/count`. I've created entries for them in the swagger file, so once master is released they should be in the spec too.
